dh_collections_prism_collection_community.json

```diff
@@ -1087,5 +1087,29 @@
   },
   "467b71cc-a252-4a43-95fb-ca5ee8124707": {
     "community_id": "food-allergy-data-dictionary"
+  },
```

Comment:
In the meeting we had this morning (29-03-2022), we noticed that these had been overlooked from the mapping JSON we created. @fenekku had other changes that have been merged and this should be the only other changes needed for now.